snowflake/changelog.d/17130.fixed

```diff
@@ -0,0 +1 @@
+Fixed Snowflake table storage metrics
```

Comment:
Based on the implementation it looks like you also added some new metrics. I think this means we need 2 changelog entries. In one you say what kind of metrics you added, and in another you say **what** you fixed about table storage metrics.